examples/MLflow-model/model/conda.yaml

```diff
@@ -0,0 +1,11 @@
+channels:
```

Comment:
We put the model examples here: https://github.com/kleveross/ormb/tree/master/examples